app-admin/conky/conky-1.11.6_pre.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A free, light-weight system monitor"
+DESCRIPTION="Conky is a free, light-weight system monitor for X, that displays \
+any kind of information on your desktop. \
```

Comment:
No " \" at the end to force the next text to be displayed on a new line, check some other lines also, no need for the backslash at the end (check after build with opening the package in HaikuDepot)